src/pages/[id].page.jsx

```diff
@@ -66,37 +68,50 @@ export default function Details({ componentName }) {
             and the answer to that question is in the Accordion panel.
           </p>
         </Definition>
-        <div className="cmp-specifications-block">
-          {/* This content will be replaced with Air Table data */}
-          <SpecificationBlock heading="Focus Expectations">
-            <RichText
-              markdown="Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod
+        {/* This content will be replaced with Air Table data */}
+        <UsageGuidelines>
+          <h3>Heading3</h3>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod
+          doloremque minima ducimus et voluptatem natus repudiandae porro vel,
+          animi fugiat. Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Quod doloremque minima ducimus et voluptatem natus repudiandae porro
+          vel, animi fugiat.
+          <h3>Heading3</h3>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod
+          doloremque minima ducimus et voluptatem natus repudiandae porro vel,
+          animi fugiat. Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Quod doloremque minima ducimus et voluptatem natus repudiandae porro
+          vel, animi fugiat.
+        </UsageGuidelines>
+
+        <SpecificationBlock heading="Focus Expectations">
+          <RichText
```

Comment:
Good job using the RichText component that just got merged 🏆

Reply:
mobile view image is now centered with [this commit](https://github.com/sparkbox/accessible-components/pull/45/commits/873cfd3698a3ce35cfafc4f6fac8f0fb38bd27a3)!
![image](https://user-images.githubusercontent.com/49820744/198671335-cab78198-f8a2-4295-b11c-1ec3f92a64cf.png)